e2e/test/TwinE2ETests.cs

```diff
@@ -92,16 +88,12 @@ public async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesEvent_WithObse
             await Twin_ServiceSetsDesiredPropertyAndDeviceReceivesEventAsync(Client.TransportType.Mqtt_WebSocket_Only, SetTwinPropertyUpdateCallbackObsoleteHandlerAsync, Guid.NewGuid().ToString()).ConfigureAwait(false);
         }
 
-        // TODO: #1132 - Twin desired property updates not received after fault injection on AMQP
```

Comment:
isn't this the same issue we noticed with the java SDK? that you will have to resend the AMQP messages when you open the links back up?

Reply:
We should probably bring the fix to the C# SDK as well

Reply:
That is exactly what this PR does, actually. The fix for the C# SDK was checked into master, and now this merge brings that same fix to preview. That's why I took the time to re-enable any tests that were only disabled due to 1132